src/data/thermal-conductivity-constants.ts

```diff
@@ -16,6 +16,369 @@ export const DIPPRThermalConductivityDictionary: Record<
     thermalConductivityAtMinimumTemperature: 0.2524,
     thermalConductivityAtMaximumTemperature: 0.1727,
   },
+  Acetamide:{
+    name: 'Acetamide',
+    formula: 'C2H5NO',
+    CAS: '60-35-5',
+    molecularWeight:  59.0672,
+    C1:  0.39363,
+    C2: -0.00037053,
+    C3:NaN,
```

Comment:
Sir @maneetgoyal  can it be optional? here in many compounds C3,C4 and C5 is empty

Reply:
Yeah, leaving empty should be convenient